src/App.tsx

```diff
@@ -1,24 +1,304 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import classNames from 'classnames';
 import { UserWarning } from './UserWarning';
+import { Footer } from './components/Footer';
+import { Header } from './components/Header';
+import { TodoList } from './components/TodoList';
+import { Todo } from './types/Todo';
+import {
+  getTodos,
+  postTodo,
+  removeTodo,
+  updateTodo,
+} from './api/todos';
+import { FilterType } from './types/FilterType';
 
-const USER_ID = 0;
+const USER_ID = 10824;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [query, setQuery] = useState('');
+  const [errorType, setErrorType] = useState('');
+  const [filterType, setFilterType] = useState(FilterType.ALL);
+  const [loading, setLoading] = useState(false);
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [indexUpdatedTodo, setIndexUpdatedTodo] = useState(0);
```

Comment:
```suggestion
  const [indexUpdatedTodo, setIndexUpdatedTodo] = useState(0);
```
for what do you need this state

Reply:
to insert tempTodo render in current place in the list when you delete it